source/includes/usage-examples/write-code-examples.php

```diff
@@ -0,0 +1,104 @@
+<?php
+require 'vendor/autoload.php'; 
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+$collection = $client->db->coll;
+
+// Inserts one document that stores the specified value
+// start-insert-one
+$result = $collection->insertOne(['<field name>' => '<value>']);
```

Comment:
S: might be good to include more than one field here
eg
```suggestion
$result = $collection->insertOne([
    '<field name 1>' => '<value 1>',
    '<field name 2>' => '<value 2>'
]);
```